Demo/src/main/res/layout/fragment_paypal_buttons_styling.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="@dimen/padding_16dp">
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/labelFunding"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:labelFor="@id/paymentButtonFundingSpinner"
+        android:text="@string/select_funding"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Spinner
+        android:id="@+id/paymentButtonFundingSpinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:prompt="@string/select_funding"
+        android:spinnerMode="dialog"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/labelFunding" />
```

Comment:
🧶  : Can we migrate to Material Design's `TextInputLayout` here to be consistent with `CardFragment`.

Reply:
`TextInputLayout` gives nice animation effects when user's typing, in this case spinner opens a dialog so it will not be visible anyway

Reply:
Got it. The interface is nearly identical to a Spinner (checking this [stack overflow post](https://stackoverflow.com/a/56152098)). It's a weird API but you get material design and the typing will be disabled with this style.